pkg/sql/sem/builtins/unaccent_dictionary.go

```diff
@@ -0,0 +1,1519 @@
+// Copyright 2015 The Cockroach Authors.
```

Comment:
nit: 2020.

I think this may be useful as a separate package, i.e. in `pkg/util/unaccent/unaccent.go`

Reply:
Ok, done